tests/homework/q6_employee_male.py

```diff
@@ -16,5 +16,18 @@
 # user: postgres
 # password: postgres
 """
+import psycopg
 def find_employee_male_table() -> list:
-    pass
+    with psycopg.connect("host=localhost dbname=postgres user=postgres password=postgres") as conn:
+        with conn.cursor() as cur:
+            cur.execute("""
+             select name, age, position
+               from employee
+              where gender= 'Male'
+              order by age desc""")
+
+            male= cur.fetchall()
```

Comment:
```suggestion
            male = cur.fetchall()
```
코딩을 할 때는 일관적인 코딩 스타일도 중요합니다.